roles/tpa_single_node/templates/manifests/init/dataset/Deployment.yaml.j2

```
@@ -0,0 +1,39 @@
+kind: Deployment
```

You need to add the following at the top :

```suggestion
apiVersion: apps/v1
```